bin/run_docker.py

```diff
@@ -64,12 +76,63 @@ def make_invalid_output(file_name: str, log_file_path: str, file_content: str) -
     return invalid_file
 
 
-def handle_outputs(output_path: str, output_file_name: str, log_text: str):
+def check_output_file_size(
+    file_list: List[str],
+    log_text: str,
+    bad_output_msg: Optional[str],
+) -> NamedTuple:
+    """
+    Checks if the given output file generated from the Docker submission is empty.
+    This function handles the following cases:
+    1. If the output file is empty, it will update and return the log_text and bad_output_msg inputs
+    2. If the output file is not empty, it will return the log_text and bad_output_msg inputs
+    as they were fed into the function call originally
+
+    Arguments:
+        file_list: The output file generated from the Docker submission (stored in a list)
+        log_text: The current log text
+        bad_output_msg: The current bad output message
+
+    Returns:
+        log_text: The updated log text (if applicable)
+        bad_output_msg: The updated bad output message (if applicable)
+
+    """
+    # Declaring variables
+    # We would only run this function if there is only one output file in the list
+    file = file_list[0]
+    incorrect_size = False
```

Comment:
NIT: If I am reading this correctly you don't need to set this default `False` value here. In your if/else you will always get a boolean value assigned to `incorrect_size`.

Reply:
You're right, the `any()` call would return True/False, and I assign `incorrect_size` as True/False in line 117. Removing. Thanks!